docs/automation.md

```diff
@@ -87,4 +87,59 @@ This is a snapshot of our automated checks at a glance.
 
 ![Continuous Deployment practices](./media/continuous_deployment_practices.png)
 
+## Lambda layer build & deploy
+
```

Comment:
important: missing prelude; explain what Lambda Layer is and why we need a pipeline.

Reply:
Added